src/states/elder/mod.rs

```diff
@@ -345,6 +345,7 @@ impl Elder {
             let our_member_connections: HashSet<_> = self
```

Comment:
This should be renamed as it's no longer just our members. What about something like `retained_connections`?